src/ui/webeiddialog.cpp

```diff
@@ -467,59 +467,59 @@ WebEidDialog::retriableErrorToTextTitleAndIcon(const RetriableError error)
     case RetriableError::SMART_CARD_SERVICE_IS_NOT_RUNNING:
-        return {tr("Smart card service is not running. Please start it."),
-                tr("Start smart card service"), QStringLiteral(":/images/cardreader.svg")};
+        return {tr("The Smart Card service required to use the ID-card does not work. Please run it."),
+                tr("Launch the Smart Card service"), QStringLiteral(":/images/cardreader.svg")};
     case RetriableError::NO_SMART_CARD_READERS_FOUND:
-        return {tr("No readers attached. Please connect a smart card reader."),
-                tr("Connect a smart card reader"), QStringLiteral(":/images/cardreader.svg")};
+        return {tr("Card reader not connected. Please connect the card reader to the computer."),
+                tr("Connect the card reader"), QStringLiteral(":/images/cardreader.svg")};
 
     case RetriableError::NO_SMART_CARDS_FOUND:
     case RetriableError::PKCS11_TOKEN_NOT_PRESENT:
         return {tr("No smart card in reader. "
                    "Please insert an electronic ID card into the reader."),
-                tr("Insert an ID card"), QStringLiteral(":/images/id-card.svg")};
+                tr("Insert the ID-card"), QStringLiteral(":/images/id-card.svg")};
     case RetriableError::SMART_CARD_WAS_REMOVED:
     case RetriableError::PKCS11_TOKEN_REMOVED:
         return {tr("The smart card was removed. "
                    "Please insert an electronic ID card into the reader."),
-                tr("Insert an ID card"), QStringLiteral(":/images/id-card.svg")};
+                tr("Insert the ID-card"), QStringLiteral(":/images/id-card.svg")};
 
     case RetriableError::SMART_CARD_TRANSACTION_FAILED:
         return {tr("The smart card transaction failed. "
                    "Please make sure that the smart card and reader are properly connected."),
-                tr("Check the ID card connection"), QStringLiteral(":/images/id-card.svg")};
+                tr("Check the ID-card and the reader connection"), QStringLiteral(":/images/id-card.svg")};
     case RetriableError::FAILED_TO_COMMUNICATE_WITH_CARD_OR_READER:
         return {tr("Failed to communicate with the smart card or reader. "
                    "Please make sure that the smart card and reader are properly connected."),
-                tr("Check the ID card connection"), QStringLiteral(":/images/id-card.svg")};
+                tr("Check the ID-card and the reader connection"), QStringLiteral(":/images/id-card.svg")};
 
     case RetriableError::SMART_CARD_CHANGE_REQUIRED:
-        return {tr("The smart card is malfunctioning, please change the smart card."),
-                tr("Change the ID card"), QStringLiteral(":/images/id-card.svg")};
+        return {tr("The desired operation cannot be performed with the inserted ID-card. Make sure that the ID-card is supported by the Web eID application."),
+                tr("ID-card is not supported"), QStringLiteral(":/images/id-card.svg")};
```

Comment:
I think that we cannot always say that the ID card is not supported in case of this error, for example, it is raised in case the e-service sends a signing hash that the card does not support. Would _Operation not supported by ID card_ be acceptable?